test/schema_test.exs

```diff
@@ -103,6 +103,10 @@ defmodule OpenApiSpex.SchemaTest do
       assert Schema.example(%Schema{type: :string, example: "foo"}) == "foo"
     end
 
+    test "uses the first value in `examples` property when not nil" do
```

Comment:
[note] Returning the first value seems consistent with how https://editor-next.swagger.io/ generates examples.